src/runtime/route-rules.ts

```diff
@@ -32,7 +33,7 @@ export function getRouteRules(event: H3Event): NitroRouteRules {
   event.context._nitro = event.context._nitro || {};
   if (!event.context._nitro.routeRules) {
     const path = new URL(event.req.url, "http://localhost").pathname;
-    event.context._nitro.routeRules = getRouteRulesForPath(path);
+    event.context._nitro.routeRules = getRouteRulesForPath(withoutBase(path, useRuntimeConfig().app.baseURL));
```

Comment:
I'm wondering if we should do this directly inside `getRouteRulesForPath`?

Reply:
I felt it was more natural here as we have access to `event`; `getRouteRulesForPath` feels like more of a 'pure' function. Also, looking at usage, it is used with an already normalised url, meaning if we move this there we are likely to double-remove a baseURL:

https://github.com/unjs/nitropack/blob/6541373545ee2716566bce46c7568ad2c3164cdf/src/runtime/app.ts#L58-L60